zip-draft-XXXX.rst

```diff
@@ -0,0 +1,269 @@
+::
+
+  ZIP: Unassigned
+  Title: Lockbox for Decentralized Grants Allocation
+  Owners: Skylar Saveland <skylar@free2z.com>
+          Jason McGee <>
+  Credits:
+  Status: Draft
+  Category: Consensus
+  Created: 2024-06-26
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/858>
+
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and
+only when, they appear in all capitals.
+
+.. {Avoid duplicating definitions from other ZIPs. Instead use wording like this:}
+
+.. The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+.. defined in the Zcash protocol specification [#protocol-networks]_.
+
+.. The term "full validator" in this document is to be interpreted as defined in
+.. the Zcash protocol specification [#protocol-blockchain]_.
+
+.. The terms below are to be interpreted as follows:
+
+.. {Term to be defined}
+..   {Definition.}
+.. {Another term}
+..   {Definition.}
+
+
+.. |percentage| replace:: 50%
+
+
+
+Abstract
+========
+
+This ZIP proposes the allocation of |percentage| of the Zcash block reward,
+post-November 2024 halving, to a decentralized "lockbox." Currently, 80% of the
```

Comment:
```suggestion
post-November 2024 halving, to an in-protocol "lockbox." Currently, 80% of the
```
The lockbox itself is a feature of the protocol; the ZIP should identify that the funds 
stored in the lockbox are intended to be distributed in the future to fund development
via a to-be-determined decentralized protocol.